README.md

```diff
@@ -34,31 +34,53 @@
 
 ## About IPFS Companion
 
-IPFS Companion harnesses the power of your locally running IPFS node (either through the [IPFS Desktop](https://docs.ipfs.io/install/ipfs-desktop/) app or the [command-line daemon](https://docs.ipfs.io/install/command-line/)) directly inside your favorite Chromium-based or Firefox browser, enabling support for [`ipfs://` addresses](https://docs.ipfs.io/how-to/address-ipfs-on-web/#native-urls), redirecting content-addressed  websites and file paths to your local [Gateway](https://docs.ipfs.io/concepts/glossary/#gateway), easy IPFS file import and sharing, and more.
+IPFS Companion harnesses the power of your locally running IPFS [Kubo](https://github.com/ipfs/kubo) node (either through the [IPFS Desktop](https://docs.ipfs.tech/install/ipfs-desktop/) app or the [command-line daemon](https://docs.ipfs.tech/install/command-line/)) directly inside your favorite Chromium-based or Firefox browser, enabling support for [`ipfs://` addresses](https://docs.ipfs.tech/how-to/address-ipfs-on-web/#native-urls), redirecting content-addressed websites and file paths to your local [Gateway](https://docs.ipfs.tech/concepts/glossary/#gateway), easy IPFS file import and sharing, and more.
 
-IPFS is a peer-to-peer hypermedia protocol designed to make the web faster, safer, more resilient, and more open. It enables the creation and dissemination of completely distributed sites and applications that don’t rely on centralized hosting and stay true to the original vision of an open, flat web. Visit [the IPFS Project website](https://ipfs.io) to learn more.
+IPFS is a peer-to-peer hypermedia protocol designed to make the web faster, safer, more resilient, and more open. It enables completely distributed sites and applications that don’t rely on centralized hosting and stay true to the original vision of an open, flat web. Visit [the IPFS Project website](https://ipfs.tech) to learn more.
```

Comment:
maybe a callout to ipshipyard.com could be placed here as well?

or "maintained by ipshipyard" somewhere?

Reply:
good idea, added in 7630978969c41615f71500506b1199c7afe533e6